src/plugins/camera/camera_definition.cpp

```diff
@@ -486,9 +486,6 @@ bool CameraDefinition::get_possible_settings(
 
     for (const auto& parameter : _parameter_map) {
         for (const auto& option : parameter.second->options) {
-            if (_current_settings[parameter.first].needs_updating) {
-                continue;
-            }
```

Comment:
I removed those lines because an exclusion does not depend on `needs_updating`. I could verify in the test camera_definition that the unit test below was wrong: it was exposing `CAM_VIDFMT` when it should actually have been excluded.